Formula/microws.rb

```diff
@@ -0,0 +1,41 @@
+class Microws < Formula
+  desc "Tiny WebSockets"
+  homepage "https://github.com/uNetworking/uWebSockets"
+  url "https://github.com/uNetworking/uWebSockets/archive/v0.14.5.tar.gz"
+  sha256 "4e4a4bcde543baae57ca8f327304874d1bb1fb11b2c4f71618d2b3279003f2d5"
+
```

Comment:
maybe include a `head` with `head "https://github.com/uNetworking/uWebSockets.git"`?

Reply:
Sometimes it's worth holding off to see what kind of demand there is for a `head` spec. They can add quite a bit of support burden sometimes; people often seem to expect Homebrew to provide support when `HEAD` builds fail but Homebrew practically speaking can't provide support for a moving target.

I defer to ILZ for an opinion here though.

Reply:
i only think it makes sense if the formula is using `github.com/user/project/archive/version`

since that's just a tar of the source, it shouldn't be a moving target (until someone makes a change that won't work for the next release anyway)

Reply:
Yeah, I meant that `HEAD` is a moving target. Something that is broken today may be fixed tomorrow, and indeed, vice versa.

Reply:
gotcha, excellent point.

i was referring to `HEAD` changes in build procedures (specifically)

Reply:
Yeah `head` is not something we want to routinely add except when someone comes asking for it for their own use.